App.tsx

```diff
@@ -0,0 +1,44 @@
+import './polyfills';
+import React from 'react';
+import { StatusBar } from 'expo-status-bar';
+import { WagmiConfig } from 'wagmi';
+import { mainnet, polygon, arbitrum, polygonMumbai } from 'wagmi/chains';
+import {
+  createWeb3Modal,
+  defaultWagmiConfig,
+  Web3Modal,
+} from '@web3modal/wagmi-react-native';
+import HomePage from './src/pages/HomePage';
+
+const projectId = process.env.EXPO_PUBLIC_WALLETCONNECT_CLOUD_PROJECT_ID;
+
+const metadata = {
+  name: 'Example dApp',
+  description: 'Modern Example dApp from Callstack',
```

Comment:
[nit] Not a native speaker but there is something with the word order
```suggestion
  description: 'Modern dApp example from Callstack',
```

Reply:
Agree